--- Perlbal-1.59/t/45-buffereduploads.t	2007-01-17 00:15:34.000000000 +0100
+++ /home/ruben/perlbal/t/45-buffereduploads.t	2007-06-20 23:26:29.000000000 +0200
@@ -83,10 +83,13 @@
         );
 
 # rate tests
-buffer_rules(rate => 700_000);
-request("buffer_on_rate", 500_000,
+buffer_rules(rate => 200_000);
+request("buffer_on_rate", 1_000_000,
         50_000,
-        "sleep:0.6",
+        "sleep:1",
+        "empty",
+        300_000,
+        "sleep:1",
         300_000,
         "exists",
         "finish",
